### ISSUE TITLE
Request to remove from blacklist

### ISSUE THREAD

**Issue by spacedingo** [quidsup/notrack#225]
Found couple of domains that render some leading websites broken:
purch.com -- no images on www.tomshardware.com
adobedtm.com -- no videos on www.redbull.tv

**Comment by mintyfreshpenguin** [comment]
Hey Spacedingo
Did you know you can whitelist sites yourself? There’s a section under config on the Dashboard.

I can’t speak for the man Quids himself. But I doubt he will remove the sites you asked for from the blacklist. And TBH I’d be rather disappointed if he did. 

Why?

AdobeDTM as I understand is a tool used by marketers to identify visitors and deliver user specific content. I imagine things like this is why most of us are using NoTrack. 

This [site](https://www.slideshare.net/JessicaSmith53/why-you-should-know-about-adobes-dynamic-tag-manager) give a good explanation of AdobeDTM

HTH

**Comment by spacedingo** [comment]
@mintyfreshpenguin I whitelisted them before I made this request.
I made sure that there had been past requests for removals ([closed issues](https://github.com/quidsup/notrack/issues?q=is%3Aissue+is%3Aclosed)).
I understand that I could use the whitelisting feature. However, blocklisting at least to me, is a balance between (breaking) function vs privacy. I simply volunteered to bring the breakage to attention : )
